src/esp/sensor/CameraSensor.cpp

```diff
@@ -15,64 +15,80 @@
 namespace esp {
 namespace sensor {
 
+CameraSensorSpec::CameraSensorSpec() : VisualSensorSpec() {
+  uuid = "rgba_camera";
+  sensorSubType = SensorSubType::Pinhole;
+}
+
+void CameraSensorSpec::sanityCheck() {
+  VisualSensorSpec::sanityCheck();
+  CORRADE_ASSERT(sensorSubType == SensorSubType::Pinhole ||
+                     sensorSubType == SensorSubType::Orthographic,
+                 "CameraSensor::CameraSensorSpec(): sensorSpec does not have "
```

Comment:
CameraSensorSpec::sanityCheck()
Please check and fix the similar errors everywhere.